src/reducers/global.ts

```diff
@@ -362,6 +362,16 @@ export const global = (state = defaultState, action: any) => {
       };
     }
 
+    case ACTION_TYPES.FETCH_ERROR_CLEAR: {
```

Comment:
Nit: This is not technically an `ERROR_CLEAR`, it's a clear of the display, more `ERROR_DISPLAY_CLEAR`. The prior `errorMessage` and `errorDuringFetch` values will remain after this reducer happens. I think that's a fine design choice -- you might want to clear the error warning being displayed but still have some part of your app know the prior fetch failed -- but the `FETCH_ERROR_CLEAR` to feels like it implies a full clear of everything about error.

Reply:
TY! Will update